database.html

```diff
@@ -0,0 +1,112 @@
+<!doctype html>
+<html lang="en">
+  <head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+	<!-- Bootstrap CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+	
+	<!-- jQuery library, then Popper JS, then JavaScript -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+	
+	<title>MyMood Database Interface</title>
+  </head>
+  
+  <body>

```

Comment:
Let's keep the indentation consistent. It's using 2 spaces here, but tabs everywhere else.

We should decide on 1 and have all our HTML pages follow the same paradigm.
What do you think of following this: https://www.w3schools.com/html/html5_syntax.asp
I'll adjust the style for the dev guidelines / user manual later to match what we decide on now

Reply:
I checked out both the links, I changed all the indentation to 2 spaces.